Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Editor/Scripts/bootstrap.py

```diff
@@ -303,5 +303,6 @@ def bootstrap_AssetBuilder():
     # custom prompt
     sys.ps1 = "[azpy]>>"
     
-_LOGGER.debug('~ DCCsi: bootstrap.py took: {} sec'.format(time.process_time() - _START)) 
+_END_TIME = time.process_time() - _START
```

Comment:
This should be replaced with timeit <-- is a better time indicator and measure then the time module.